src/modsim2/model/resnet.py

```diff
@@ -0,0 +1,108 @@
+# Modified from mod sim phase 1 code
+
+import pytorch_lightning as pl
+import torch
+import torch.nn.functional as F
+import torchvision
+from torchmetrics import Accuracy
+
+
+def create_model(num_classes=10, channels=3):
+    """
+    Function to create a resnet18 model for CIAFAR-10.
+
+    Args:
+        num_classes: Number of classes in the dataset
+        channels: Number of channels in the image
+
+    Returns: ResNet18 model
+    """
+    model = torchvision.models.resnet18(pretrained=False, num_classes=num_classes)
```

Comment:
<strike>maybe make `pretrained` an arg we can pass in?</strike>
actually, we'd then mess this up with our modification to conv1... maybe worth adding a comment stating that we're deliberately not allowing pretrained here

Reply:
Have done this